From e5be88f83b8fc2e95d8479388b5a4749217551f7 Mon Sep 17 00:00:00 2001
From: Phantom X <PhantomX@users.noreply.github.com>
Date: Mon, 18 Apr 2022 09:37:58 -0300
Subject: [PATCH] CSD: Prefer SSD by default

This fixes usage on Plasma Wayland when GTK_CSD=0 and don't break others
---
 gtk/gtkwindow.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gtk/gtkwindow.c b/gtk/gtkwindow.c
index 05ed4e0..ba71f29 100644
--- a/gtk/gtkwindow.c
+++ b/gtk/gtkwindow.c
@@ -3839,7 +3839,7 @@ gtk_window_should_use_csd (GtkWindow *window)
   if (GDK_IS_WAYLAND_DISPLAY (gtk_widget_get_display (GTK_WIDGET (window))))
     {
       GdkDisplay *gdk_display = gtk_widget_get_display (GTK_WIDGET (window));
-      return !gdk_wayland_display_prefers_ssd (gdk_display);
+      return gdk_wayland_display_prefers_ssd (gdk_display);
     }
 #endif
 
-- 
2.35.1

